refresh.template.py

```diff
@@ -147,6 +148,17 @@ def _get_headers_msvc(compile_args: typing.List[str], source_path: str):
         # End:   template filled by Bazel
     ))
 
+    # Write header_cmd to a temporary file, so we can use it as a parameter to cl.exe,
+    # because Windows cmd has a limitation of 8KB of command line length. So here we make a threshold of len(compile_args) < 8000
+    WIN_CMD_LIMITS = 8000
+    if len('\n'.join(header_cmd)) >= WIN_CMD_LIMITS:
+        fd, temp_params = tempfile.mkstemp(text=True)
+        with open(temp_params, 'w') as f:
+            # should skip cl.exe the 1st line
+            f.write('\n'.join(header_cmd[1:]))
```

Comment:
Unsure: Do we need a newline at the end?

Reply:
No need for cl.exe to work, but a newline at end is better to print and read.